main/src/ca/uwaterloo/flix/language/phase/Lowering.scala

```diff
@@ -1573,6 +1577,23 @@ object Lowering {
     LoweredAst.Expression.Tuple(exps, tpe, pur, eff, loc)
   }
 
+  /**
+    * Returns an InvokeStaticMethod expression to compile the regex pattern `exp`.
+    */
+  private def mkRegex(patt: String, loc: SourceLocation): LoweredAst.Expression = {
+    val cls = Class.forName("java.util.regex.Pattern")
+    val mtd = cls.getDeclaredMethod("compile", Class.forName("java.lang.String"))
+    val exp = LoweredAst.Expression.Cst(Ast.Constant.Str(patt), Type.Str, loc)
+    println(mtd)
+    // val tpe = Type.mkNative(cls, loc)
```

Comment:
I think this should be the return type, i.e. `Type.mkNative` of `cls`.

Reply:
Removed - Regex is now a type like BigDecimal.